#!/usr/bin/env python
PACKAGE = "path_track"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sample_time",double_t,0,"double parameter",0.1,0,2)
gen.add("max_vel",double_t,0,"double parameter",2,0,5)
gen.add("max_w",double_t,0,"double parameter",5,0,15)
gen.add("max_vel_acc",double_t,0,"double parameter",2,0,15)
gen.add("max_w_acc",double_t,0,"double parameter",2,0,15)
gen.add("a",double_t,0,"double parameter",0.05,0,2)
gen.add("b",double_t,0,"double parameter",0.2,0,2)
gen.add("c",double_t,0,"double parameter",0.1,0,2)
gen.add("R_inflation",double_t,0,"double parameter",0.5,0,5)
gen.add("dis_thre",double_t,0,"double parameter",0.2,0,5)

exit(gen.generate(PACKAGE,"path_track","DWA_"))
